README.md

```diff
@@ -80,61 +81,72 @@ The current version cannot handle plists or dictionaries.
 Simple example that uses the `com.apple.systempreferences` domain,
 with a single key and value.
 
-    mac_os_x_userdefaults "enable time machine on unsupported volumes" do
-      domain "com.apple.systempreferences"
-      key "TMShowUnsupportedNetworkVolumes"
+    mac_os_x_userdefaults 'enable time machine on unsupported volumes' do
+      domain 'com.apple.systempreferences'
+      key 'TMShowUnsupportedNetworkVolumes'
       value "1"
     end
 
 Specify a global domain. Note that the key is not required for global domains.
 
-    mac_os_x_userdefaults "full keyboard access to all controls" do
-      domain "AppleKeyboardUIMode"
+    mac_os_x_userdefaults 'full keyboard access to all controls' do
+      domain 'AppleKeyboardUIMode'
       global true
-      value "2"
+      value '2'
     end
 
 A boolean type that uses truthiness (TRUE).
 
-    mac_os_x_userdefaults "finder expanded save dialogs" do
-      domain "NSNavPanelExpandedStateForSaveMode"
+    mac_os_x_userdefaults 'finder expanded save dialogs' do
+      domain 'NSNavPanelExpandedStateForSaveMode'
       global true
-      value "TRUE"
-      type "bool"
+      value 'TRUE'
+      type 'bool'
     end
 
 A setting that uses an int (integer) type.
 
-    mac_os_x_userdefaults "enable OS X firewall" do
-      domain "/Library/Preferences/com.apple.alf"
-      key "globalstate"
-      value "1"
-      type "int"
+    mac_os_x_userdefaults 'enable OS X firewall' do
+      domain '/Library/Preferences/com.apple.alf'
+      key 'globalstate'
+      value '1'
+      type 'int'
+    end
+
+Disable Screen Saver for particular user (uses '-currentHost' flag).
+
+    mac_os_x_userdefaults 'Disable Screen Saver' do
```

Comment:
I appreciate all of the changes in this branch and the addition of host, but can we provide an example of when we would use the host flag? From my understanding, it is not required for this example to work as expected.